jena/src/test/scala/org/w3/banana/jena/JenaSerialisationTest.scala

```diff
@@ -1,11 +1,42 @@
 package org.w3.banana.jena
 
-import org.w3.banana.io.{NTriplesReaderTestSuite, RdfXMLTestSuite, TurtleTestSuite}
+import org.w3.banana.Prefix
+import org.w3.banana.io.{NTriplesReaderTestSuite, RdfXMLTestSuite, TurtleTestSuite, RDFWriter}
 import scala.util.Try
 import org.w3.banana.util.tryInstances._
+import scalaz.syntax._, comonad._
 
-class JenaTurtleTest extends TurtleTestSuite[Jena, Try]
+class JenaTurtleTest extends TurtleTestSuite[Jena, Try] {
+  "write with prefixes" in {
+    val prefix = Prefix[Jena]("foo", "http://purl.org/dc/elements/1.1/")
+
+    val sout = writer.asString(referenceGraph, "", prefix).copoint
+//    val expected =
+//      """@prefix foo:   <http://purl.org/dc/elements/1.1/> .
+//        |
+//        |<http://www.w3.org/2001/sw/RDFCore/ntriples/>
+//        |        foo:creator    "Dave Beckett" , "Art Barstow" ;
+//        |        foo:publisher  <http://www.w3.org/> .""".stripMargin
+//    sout must_== expected // TODO this doesn't work as expected for some reason
+
+    val l: Iterator[String] = sout.lines
+    l.next() must_== """@prefix foo:   <http://purl.org/dc/elements/1.1/> ."""
+    l.next() must_== """"""
+    l.next() must_== """<http://www.w3.org/2001/sw/RDFCore/ntriples/>"""
+    l.next() must_== """        foo:creator    "Dave Beckett" , "Art Barstow" ;"""
+    l.next() must_== """        foo:publisher  <http://www.w3.org/> ."""
+
```

Comment:
Is there a way of making this a bit less fragile? I can just imagine that different serialisers end up having different ways of writing out the same RDF, and even the same serialiser may have different results at different times. For example, there is no reason that `"Dave Beckett"` appears before `"Art Barstow"` or that `creator` appears first before `publisher`. Nor is it necessary that one serializer uses the same number of white spaces as another one.